cosmo_tester/test_suites/cluster/conftest.py

```diff
@@ -240,7 +241,8 @@ def nine_vms(nine_session_vms, test_config, logger):
                      broker_count=3, db_count=3,
                      manager_count=3, bootstrap=False)
     for vm in nine_session_vms:
-        vm.teardown()
+        _remove_cluster(vm, logger)
+        vm.teardown(kill_certs=False)
```

Comment:
Certs should be killed, there shouldn't be relation required between the certs of different tests.